app/forms/hub/create_client_form.rb

```diff
@@ -61,6 +61,27 @@ def save(current_user)
         tax_returns_attributes: @tax_returns_attributes.map { |_, v| create_tax_return_for_year?(v[:year]) ? tax_return_defaults.merge(v) : nil }.compact
       )
 
+      locale = @client.intake.preferred_interview_language == "es" ? "es" : "en"
+      if @client.intake.sms_notification_opt_in == "yes"
```

Comment:
Eventually we should consolidate this logic so that we don't have to do so much setup. something like @client.send_message(), where the method takes care of determining which form to send it in. There's a lot of duplicated logic across the app we'd be able to clean up! No action necessary now, just something to think about.